### PR TITLE
change-deprecated-examples Remove deprecated functions from our examples

### PR DESCRIPTION
**Pull request by stephenlawrence**

### CODE REVIEW

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ name: "Application"
 properties:
   logo_url: http://www.replicated.com/images/logo.png
   console_title: Application
- 
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:28]: probably shouldnt strip trailing whitespace in yaml

File: docs/kb/developer-resources/certs-and-keys.md [hunk 2]
```diff
@@ -6,26 +6,26 @@ weight = "999999"
...
 
 ```yml
-# The meta-data about your application 
+# The meta-data about your application
 replicated_api_version: 1.3.0
 properties:
   app_url: http://{{repl ConfigOption "hostname" }}
   console_title: Flask App
-  
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/certs-and-keys.md:28]: ditto

File: docs/kb/developer-resources/certs-and-keys.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ config:
       value_at: 1
     type: file
     affix: right
-    
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/certs-and-keys.md:76]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 4]
```diff
@@ -49,14 +49,14 @@ components:
           - host_path: /keys
             container_path: /keys
             is_ephemeral: true
-            
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:52]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 5]
```diff
@@ -49,14 +49,14 @@ components:
...
         cmd: '["/create_ssh_key.sh"]'
         config_files:
           - filename: /create_ssh_key.sh
             file_mode: 0700
             contents: |
               #!/bin/bash
-              
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:59]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 6]
```diff
@@ -65,19 +65,19 @@ components:
                   exit 0
                 fi
               fi
-              
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:68]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 7]
```diff
@@ -65,19 +65,19 @@ components:
...
               echo "Generate a new key, if it does not exist"
               if [ ! -f /keys/id_rsa.pub ]; then
                 echo "Install the ssh-keygen command"
                 apt-get update && apt-get install -y keychain && rm -rf /var/lib/apt/lists/*
-              
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:73]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 8]
```diff
@@ -65,19 +65,19 @@ components:
...
                 echo "Create an ssh key"
                 ssh-keygen -f /keys/id_rsa -t rsa -N ''
               fi
-              
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:77]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 9]
```diff
@@ -65,19 +65,19 @@ components:
...
               echo "Write the public key to the authorized keys file"
               cat /keys/id_rsa.pub >> /.ssh/authorized_keys
-  
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:80]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 10]
```diff
@@ -105,14 +105,14 @@ components:
 
               ## It would be preferrable to create a custom image with this next line already installed.
               apt-get update && apt-get install -y ssh && rm -rf rm -rf /var/lib/apt/lists/*
-              
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:108]: here

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 11]
```diff
@@ -105,14 +105,14 @@ components:
...
               # SSH to the host and then execute the "run_on_host.sh" script on the host
-              ssh -i /keys/id_rsa -o StrictHostKeyChecking=no root@{{repl ThisHostPrivateIpAddress}} 'bash -s' < /run_on_host.sh
+              ssh -i /keys/id_rsa -o StrictHostKeyChecking=no root@{{repl ThisNodePrivateIPAddress}} 'bash -s' < /run_on_host.sh
           - filename: /run_on_host.sh
             file_mode: 0700
             contents: |
               #!/bin/bash
-              
+
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md:115]: here

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: docs/kb/developer-resources/running-commands-on-the-host-os.md (hunk 1 above)
Comment (dexhorthy) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:28]: curious why? does it affect formatting?

File: docs/kb/developer-resources/running-commands-on-the-host-os.md [hunk 12]
```diff
@@ -122,7 +122,7 @@ components:
               my_config_val='{{repl ConfigOption "my_config_option"}}'
 
               ## Complete example:
-              if [ "{{repl ConfigOption \"update_management_console\" }}" -eq "true" ]; then
+              if [ '{{repl ConfigOption "update_management_console" }}' -eq "true" ]; then
```
Comment (emosbaugh) [inline docs/kb/developer-resources/running-commands-on-the-host-os.md]: i know you didnt write this but `-eq` is reserved for integer expressions. wanna fix the bug?

File: docs/kb/developer-resources/running-commands-on-the-host-os.md (hunk 1 above)
Comment (emosbaugh) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:28]: possibly this one is a bad example. i went through this quickly. more concerned about the ones in block text

File: docs/kb/developer-resources/running-commands-on-the-host-os.md (hunk 5 above)
Comment (emosbaugh) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:59]: @dexhorthy like this one

File: docs/kb/developer-resources/running-commands-on-the-host-os.md (hunk 1 above)
Comment (stephenlawrence) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:28]: hrm, my Atom editor did that?

Comment (stephenlawrence) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:28]: The changes to the block text work ok in the yaml editor as well as in the generated file inside the container.

File: docs/kb/developer-resources/running-commands-on-the-host-os.md (hunk 5 above)
Comment (dexhorthy) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:59]: Does this break it? I would think Yaml parser would continue as long as you stayed at that indentation level.

Comment (stephenlawrence) [reply docs/kb/developer-resources/running-commands-on-the-host-os.md:59]: I was able to leave that line empty, but the lines with text I had to indent.

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**